frontend/libs/studio-components-legacy/src/components/StudioDecimalInput/StudioDecimalInput.tsx

```diff
@@ -22,7 +23,14 @@ export type StudioDecimalInputProps = Override<
 
 export const StudioDecimalInput = forwardRef(
   (
-    { description, onChange, value, validationErrorMessage, ...rest }: StudioDecimalInputProps,
+    {
+      description,
+      onBlurNumber,
+      onChange,
+      value,
+      validationErrorMessage,
+      ...rest
+    }: StudioDecimalInputProps,
```

Comment:
Vi må få med `onBlur` her også. Ellers vil den overskrive `onBlur` lengre nede siden den kommer inn som en del av `...rest`, og `onBlurNumber` vil ikke ha noen effekt hvis konsumenten har tatt i bruk begge proppene.
```suggestion
    {
      description,
      onBlur,
      onBlurNumber,
      onChange,
      value,
      validationErrorMessage,
      ...rest
    }: StudioDecimalInputProps,
```